config/environments/test.rb

```diff
@@ -1,3 +1,6 @@
+# 1 Apr 2020 [AMH] : Deprecation warnings have been VERY verbose lately. This hides them.
+Warning[:deprecated] = false
```

Comment:
Can we set an ENV variable or something else to see these again locally so that we can start addressing them? I worry that outright hiding them globally might lead to some confusion when we get around to upgrading and those warnings become errors.